composer/datasets/dataloader.py

```diff
@@ -93,7 +93,7 @@ def unwrap_data_loader(dataloader: DataLoader) -> DataLoader:
 class DataloaderHparams(hp.Hparams):
     """Hyperparameters to initialize a :class:`~torch.utils.data.Dataloader`.
 
-    Parameters:
+    Args:
```

Comment:
[nit] You may want to hide the common attributes such as `Any`, `Batch`, `Callable`, etc. by specifying a `__all__` at the top of the module.